ci-operator/config/openshift-assisted/cluster-api-provider-openshift-assisted/openshift-assisted-cluster-api-provider-openshift-assisted-master.yaml

```diff
@@ -11,16 +11,16 @@ build_root:
       # latest version compatible with most kustomize manifests without throwing warnings (v4)
       # from v5 warning will be thrown, however the ansible module will fail
       ENV KUBECTL_VERSION=v1.24.17
-      RUN dnf install -y python3-pip diffutils && \
-      pip install ansible ansible-lint jinja2-cli && \
-      dnf remove -y golang && \
+      RUN dnf remove -y golang && \
```

Comment:
FWIW this was the problem.

removing golang was removing diffutils as an "unused dependency"

So we need to do this first, then install diffutils after to ensure it stays.